charts/backstage/templates/route.yaml

```diff
@@ -2,26 +2,36 @@
 apiVersion: route.openshift.io/v1
```

Comment:
Honestly I don't have a strong opinion either way. With or without `Route` support is fine with me. I think allowing user to choose between `Route` and `Ingress` is fine, however they serve the same purpose, so I don't really mind.

@serenamarie125 , @christophe-f or @raffaelespazzoli do you have a preference? Should we support `Route`s or are we fine with `Ingress` only (as it is present in the upstream chart).

Reply:
plus 1 to allowing the choice

Reply:
@sabre1041 are you ok with keeping the option? During the last call it seemed like the majority leaned towards the option.

Reply:
I am ok with the route option. But feel we should also include parameters for `caCertificate`, `certificate`, `destinationCACertificate` and `key` as some of the termination types need those to be provided

Reply:
@sabre1041 Updated. I've tried to follow OCP docs to offer as much meaningful configuration as possible. PTAL :slightly_smiling_face:

I'm omitting the `alternateBackends` since the upstream Helm chart exposes single service/deployment anyways.